R/calcIDataConsEneBranch.R

```diff
@@ -0,0 +1,93 @@
+#' calcIDataConsEneBranch
+#'
+#' Use data to derive OPENPROM input parameter iDataConsEneBranch
```

Comment:
a better description of the data source is needed. also of the nature of the output data that the function returns